packages/ckeditor5-dev-tests/lib/utils/manual-tests/compilehtmlfiles.js

```diff
@@ -22,8 +22,10 @@ const writer = new commonmark.HtmlRenderer();
 const viewTemplate = fs.readFileSync( path.join( __dirname, 'template.html' ), 'utf-8' );
 
 module.exports = function compileManualTestHtmlFiles( buildDir, manualTestPattern ) {
-	const sourceMDFiles = globSync( path.join( manualTestPattern, '*.md' ) );
-	const sourceDirs = uniq( sourceMDFiles.map( file => path.dirname( file ) ) );
+	const sourceMdFiles = globSync( path.join( manualTestPattern, '*.md' ) );
```

Comment:
And what with 3- and 4- letter long? is there a strict rule about this that we're using?